### DISCUSSION TITLE
Poor Design Choice of New Portal Editor - Publish Button Placement

### DISCUSSION THREAD

**Discussion by kevinpearce** [Azure/api-management-developer-portal#2628] [Feature request]
## Bug description

The new Editor Layout has placed the PUBLISH button directly underneath the CLOSE button used during editing.

This would be very easy to select in error and trigger and unwanted push to a Production Service.

## Reproduction steps

1. Go to the Editor
2. Edit a Page
3. Top Right - Click Close
4. Mouse over is immediately over PUBLISH button which could be selected in error
5. PUBLISH function has no Yes/No warning so immediate PUBLISH of Portal changes

## Expected behavior

PUBLISH button may be better placed in the Side Bar menu?
Or, at the very least provide a Yes/No on click of Publish

**Comment by sthirthala** [comment]
Thanks for feedback

**Comment by madhancock** [comment]
I run victim to this every time I use it....

**Comment by kevinpearce** [comment]
To further my original point (targeted at the Publish button)

There is definitely some work to do here regarding User Experience - the newly designed Portal Editor just makes life difficult in general

Example, Close/Delete/Publish etc overlap):

![Screenshot 2024-07-29 102418](https://github.com/user-attachments/assets/2c93199e-3e48-4275-bf03-1fd6035c2bda)
